identity/src/main/java/com/android/identity/ProgressListener.java

```diff
@@ -0,0 +1,5 @@
+package com.android.identity;
+
+public interface ProgressListener {
```

Comment:
nit: Please add some sort of javadoc description. It doesn't have to be much, but an indicator of what this interface is expected to be used for would be handy. It's not quite "self-documenting" to me.